src/Common/src/Interop/Windows/mincore/Interop.OpenCommPort.cs

```diff
@@ -9,9 +9,9 @@
 
 internal partial class Interop
 {
-    internal partial class KernelBase
+    internal partial class mincore
```

Comment:
should this be capitalized?

Reply:
I am actually following what is already in the sources (about 4 other source files). If there is consensus in this regard I can take the opportunity to rename them all (scope of the change increases to System.Memory).

Reply:
I see. I was just wondering, I thought our guideline was to have classes capitilized but maybe this are exceptions as they are interop classes. cc: @stephentoub